{{ cookiecutter.__project_name_kebab_case }}/pyproject.toml

```diff
@@ -83,6 +83,7 @@ priority = "explicit"
 
 [tool.commitizen]  # https://commitizen-tools.github.io/commitizen/config/
 bump_message = "bump: v$current_version → v$new_version"
+hooks_pre_bump = ["uv lock --offline"]
```

Comment:
You'll need either `poetry lock` here, or nothing.

```suggestion

```

Reply:
Ah, good catch. It actually wasn't a problem with Poetry, so best to remove the line here.